cypress/support/cypress-support/src/utils.js

```diff
@@ -658,27 +658,60 @@ global.resolveDeviceVariable = function (key) {
  * fireLog.deepEqual(actual, expected, "deepEqual message");
  */
 
-class FireLog {
+class FireLog extends Function {
   constructor() {
-    if (!FireLog.instance) {
-      FireLog.instance = this;
-    }
+    // Creating the function body dynamically
+    const functionBody = `
+      return function (...args) {
+        return this.log(...args);
+      }
+    `;
+    super('...args', functionBody);
+
+    const handler = {
+      apply: function (target, thisArg, argumentsList) {
+        let message;
+        const methodName = target.name;
+        if (methodName.toLowerCase().includes('equal'))
```

Comment:
Having 'to be' just for `equal` might not be scalable. Since this condition is only for equal, what about assert.includes or assert.notIncludes or any new assertion? 

How about
if(more than 2 args) // for equal, include, deepEqual, strictEqual etc
message = argumentsList[argumentsList.length - 1] + 'Actual : ' + argumentsList[x] + ' Expected: ' + argumentsList[y]

else // for isNull, isDefined etc
message = argumentsList[argumentsList.length - 1] + 'Actual : ' + argumentsList[x]

Reply:
updated, but I think for include and equal messages should be different 
I set for include - `["Error code -32400 expected to be present in the list of [-32602,-32400,-40300]"]`
for other methods with more than 2 args `["Method Content validation expected true to be true"]`

Reply:
Can we make this into `Expected error code -32400 to be in [-32602,-32400,-40300]` for overall consistency with other logs?